src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -274,7 +276,7 @@ public String getAwsSecretKey() {
     }
 
     public String getAwsSessionToken() {
-        return getString("aws.session.token");
+        return getString("aws.session.token", "");
```

Comment:
Why this change?

Reply:
I was unable to start the application when I didn't set this parameter. It shouldn't be required right?

Reply:
We usually want those parameters to be explicitly set in your config file rather than java code defaulting.  But if you are not using aws, maybe we should not require this as a required config.

Reply:
I am using AWS but not a session token. Just permanent API keys and secrets. In that scenario I don't think think should be required.

Reply:
That's reasonable.